API Generator: Add missing type for integer fields in input type (#2225)
vivid-planet/comet: .changeset/shy-jars-float.md

---
"@comet/cms-api": patch
---

> Fix api-generator missing return-type function for Int fields in input-type

Reviewer: ```suggestion
API Generator:  Add missing return type function for integer fields in input type
```

> API Generator: Add missing return type function for integer fields in input type

Reviewer 2: ```suggestion
API Generator: Add missing type for integer fields in input type
```

> API Generator:  Add missing return type function for integer fields in input type

Reviewer: ```suggestion
API Generator: Add missing return type function for integer fields in input type
```
